netket/operator/_graph_operator.py

```diff
@@ -20,6 +22,23 @@
 from ._local_operator import LocalOperator
 
 
+def check_acting_on_subspace(acting_on_subspace, hilbert, graph):
```

Comment:
can you add a short docstring?

Reply:
```suggestion
def check_acting_on_subspace(acting_on_subspace, hilbert, graph):
    """Check `acting_on_subspace` argument used by various operators."""
```

Reply:
What the inputs should look like is described in the public operator docstrings that use this funtion internally. So I think a concise docstring is fine here.